### DISCUSSION TITLE
https://github.com/xfactor-toml/rust-coding-challenges/tree/master/medium/short-palindrome

### DISCUSSION THREAD

**Discussion by soldev-comai** [xfactor-toml/rust-coding-challenges#22] [Q&A] [answered]
My solution returns `out of time limit`.

What algorithm did you use?

**Comment by xfactor-toml** [comment, marked as answer]
Dynamic Planning Algorithm.

Some calls this algorithm as dynamic programming.

I created table for letter i, ij, ijj with one time search to reduce the recursive time.
